keda/values.yaml

```diff
@@ -277,6 +277,8 @@ rbac:
 
   # -- Whether RBAC for configured CRDs that can have a `scale` subresource should be created
   enabledCustomScaledRefKinds: true
+  # -- Customize the namespace for metrics-server
```

Comment:
We should highlight this is Kubernetes Metrics Server and not KEDA Metrics Server